Source/Extensions/Blazorise.Components/TransferList.razor.cs

```diff
@@ -227,6 +227,11 @@ public bool IsMoveAllStartDisabled
             ? SelectedItemsEnd.IsNullOrEmpty()
             : true;
 
+    /// <summary>
+    /// Gets a value indicating whether the move all buttons are visible.
+    /// </summary>
+    protected bool IsShowMoveAll => ShowMoveAll && SelectionMode == ListGroupSelectionMode.Multiple;
```

Comment:
Up to the developer whether he he shows the buttons for move all in single mode or not.
Otherwise you end up in a case where he wants this use case, and cannot.

Reply:
Like I said. Up to the developer whether he wants these buttons to show or not by just setting the IsMoveAll flag, no need for us to try to be smart about it. In my opinion.

If he wants the MoveAll buttons + single mode he cannot, with this change.

Reply:
> In single mode, the user should be able to select only one item, max.

The mode setting pertains the user selection. I don't see an issue with user being able to only select one item at a time. But being able to click the move all buttons if he so wishes.

Again, up to the developer how he wants to define it. I think this change is just us trying to be smart about it, and applying a needless limitation.

Reply:
Suppose a scenario in which the user must select their City of origin from the list. Only one to proceed. With current implementation nothing stops him from selecting multiple Cities.

Reply:
> Suppose a scenario in which the user must select their City of origin from the list. Only one to proceed. With current implementation nothing stops him from selecting multiple Cities.

IT's up to the developer to configure the damn component accordingly to the requirement, what the heck? Why would he have ShowMoveAll set to true, if the user can only select one??? :D

Reply:
Well I voiced my thoughts. 
You guys win by majority. You both think otherwise.

---

If you guys go with this change, I'd at least make it explicit in the parameter doc, that the buttons won't show on single mode **EVER**.
And if you want to limit more, you can also apply the limit to the MoveAll methods.

Reply:
But, if developer wants user to be able to move only one element than that doesn't have sense, this adjusment make devwloper to be able for example( hey I want my user to be able to select one element, okay nice put the single mode on, if not he will set multiple selection mode) do not delete my smartest work of all time🤣🤣🤣...

Reply:
> When we have SelectionMode="ListGroupSelectionMode.Single" the user should be able to select only one item and to also be able to move only one item

I disagree with this bit:
> and to also be able to move only one item

Nothing says that the api should limit the move to only one time. Api is called `SelectionMode` after all. Not `MoveMode` or something like that. The api states that the selection mode is SINGLE. One item per list at a time.

Reply:
@stsrki I see that you've reverted the change. 
I do agree it might be a "dumb" requirement, but the flexibility is there, we can however modify our docs single selection examples be **our suggested usage**. `ShowMoveAll=false`.

Reply:
> @stsrki I see that you've reverted the change

Yeah, if we're going to implement `CanMove*` callbacks this is not needed anymore.